### ISSUE TITLE
[amp-date-display] Allow reference template by id via the template attribute

### ISSUE THREAD

**Issue by eduardogoncalves** [ampproject/amphtml#22524]
I have an amp-list with some [nested templates](https://amp.dev/documentation/components/amp-mustache#nested-templates) that I reference by id via the template attribute on the component.

But AMP Validator fails when I try to do it in amp-date-display

```html
<amp-date-display template="dateTime" locale="pt-br" datetime="{{dataCadastro}}" layout="flex-item"> </amp-date-display>

<template type="amp-mustache" id="dateTime">
    {{day}} de {{monthName}} {{year}}<small class="fg-muted block">{{hourTwoDigit}}:{{minuteTwoDigit}}</small>
</template>
```
![image](https://user-images.githubusercontent.com/4308648/58448399-cd2a4680-80d5-11e9-8da9-4babe89654cb.png)

Is it possible to enable **template attribute** on the amp-date-display component?

**Comment by cathyxz** [comment]
Closed by https://github.com/ampproject/amphtml/pull/23866.
